pkg/resolver/searchHelper.go

```diff
@@ -0,0 +1,330 @@
+package resolver
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/doug-martin/goqu/v9"
+	"github.com/doug-martin/goqu/v9/exp"
+	"github.com/lib/pq"
+	"github.com/stolostron/search-v2-api/graph/model"
+	"github.com/stolostron/search-v2-api/pkg/config"
+	"github.com/stolostron/search-v2-api/pkg/rbac"
+	"k8s.io/klog/v2"
+)
+
+func GetPropertyTypeCache(ctx context.Context, refresh bool) (map[string]string, error) {
```

Comment:
Function scope. Most of these functions should be private (lowercase) to the resolver package.  Only make public if the function is needed from a different package.